<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск по имени</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Напиши здесь</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>410</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>250</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить информацию в файл</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_SEX">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText" stdset="0">
     <string>Sex</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="CB_YEAR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>1970</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText" stdset="0">
     <string>year</string>
    </property>
    <item>
     <property name="text">
      <string>1970</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1980</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1990</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_sex">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sex</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_year">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_TYPE">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Персона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сторудник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Студент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Преподаватель</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>55</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип</string>
    </property>
   </widget>
   <widget class="MsgBox" name="message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>289</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MsgBox</class>
   <extends>QTextBrowser</extends>
   <header>msgbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
 </slots>
</ui>
